gui/GameConqueror.py

```diff
@@ -1009,7 +1042,10 @@ def scan_thread_func(self, cmd):
         # enable set of widgets interfering with the scan
         for wid in self.disablelist:
             wid.set_sensitive(True)
-
+            
+        # Should be safe. Right? I thought this was causing a crash, but I think it's something else memory usage related.
```

Comment:
Yep, it used to fail because it was out of the `Gdk.threads` guard (I even crashed the python interpreter via some deadlock combo). Here it's fine.